static/app/types/hooks.tsx

```diff
@@ -56,6 +56,7 @@ export type HookName = keyof Hooks;
  */
 export type RouteHooks = {
   'routes:api': RoutesHook;
+  'routes:gs-root-routes': RoutesHook;
```

Comment:
If my memory serves, we had one like this before. I’m not at my computer yet, but could you go through the git history and see what it was called before?

either way we should remove the “gs-“ prefix from this. Hooks should not try to be highly specific to getsentry when possible

Reply:
Sounds good to me!